### PR TITLE
Add Train method

### PR DESCRIPTION
**Pull request by athul**
Also added a Test. Don't know if it's correct. 
The `Learn` and `Train` functions seemed similar so reused some of the code from `Learn`. 

What do you say @joicemjoseph

### CODE REVIEW

**Comment by joicemjoseph** [comment]
I suggest to add test cases for multiple languages.

File: libvarnam_test.go [hunk 1]
```diff
@@ -69,3 +69,8 @@ func TestDestroy(t *testing.T) {
 	varnam := initVarnam("hi")
 	varnam.Destroy()
 }
+func TestTrain(t *testing.T) {
+	varnam := initVarnam("ml")
+	err := varnam.Train("chrome", "ക്രോം")
+	require.NoErrorf(t, err, "Failed to perform learn. Got error: %s", err.Error())
```
Comment (joicemjoseph) [inline libvarnam_test.go:75]: throwing fatal error, change to
```go
require.NoError(t, err, "Failed to perform learn.")```

**Review by joicemjoseph — CHANGES_REQUESTED** [review]

File: libvarnam_test.go (hunk 1 above)
Comment (athul) [reply libvarnam_test.go:75]: I still changed it even then, the tests end up as fatal. I removed my part of the code and it still persists. Even ran tests as single tests, the error persists

**Review by joicemjoseph — APPROVED** [review]

**Comment by athul** [comment]
Merging This :zap:

**Pull request merged**